src/uri.rs

```diff
@@ -44,15 +44,21 @@ impl TryFrom<&str> for Uri {
     type Error = anyhow::Error;
 
     fn try_from(s: &str) -> Result<Self> {
+        // We want to check for internal links
+        // For example., links to other internal Markdown
+        // Example input: '@/internal.md'
+        let is_internal_link = s.ends_with(".md");
```

Comment:
We have to cover a few more cases here:

* [`@/pages/about.markdown`](https://www.getzola.org/documentation/content/linking/)
* [`@/pages/about.md#example`](https://www.getzola.org/documentation/content/linking/)

These are just two from the top of my head. Probably more cases will come up in the future. I suggest to create a helper function 

```
fn is_internal_link(link: &str) -> bool
```

and add some unit tests for these cases. Then we can just use that function here.

Reply:
Agreed. Where is the good place to put this function? Have got an idea?

Reply:
Since it's just a helper function you can put it into `src/uri.rs` as well. Just make sure that it's outside of the Uri struct as it's independent of that (to avoid close coupling).

Reply:
I'll look into this tonight or early tomorrow!

Reply:
Added additional tests and few more cases